src/_gettsim/transfers/elterngeld.py

```diff
@@ -1,347 +1,256 @@
-    geburtsjahr: int,
+def elterngeld_anspruchsbedingungen_erfüllt(  # noqa: PLR0913
+    elterngeld_claimed: bool,
+    arbeitsstunden_w: float,
+    kind_anspruchsberechtigt_fg: bool,
+    vorjahr_einkommen_unter_bezugsgrenze: bool,
+    monate_elterngeldbezug_unter_grenze_fg: bool,
     elterngeld_params: dict,
 ) -> bool:
-    """Check for sibling bonus on parental leave benefit.
-
-    # ToDo: why use datum and geburtsjahr instead of alter?
+    """Individual is eligible to receive Elterngeld.
 
     Parameters
     ----------
-    geburtsjahr
-        See basic input variable :ref:`geburtsjahr <geburtsjahr>`.
+    elterngeld_claimed
+        See basic input variable :ref:`elterngeld_claimed <elterngeld_claimed>`.
+    arbeitsstunden_w
+        See basic input variable :ref:`arbeitsstunden_w <arbeitsstunden_w>`.
+    kind_anspruchsberechtigt_fg
+        See :func:`kind_anspruchsberechtigt_fg`.
+    vorjahr_einkommen_unter_bezugsgrenze
+        See :func:`vorjahr_einkommen_unter_bezugsgrenze`.
+    monate_elterngeldbezug_unter_grenze_fg
+        See :func:`monate_elterngeldbezug_unter_grenze_fg`.
     elterngeld_params
         See params documentation :ref:`elterngeld_params <elterngeld_params>`.
 
     Returns
     -------
 
     """
-    current_year = elterngeld_params["datum"].astype("datetime64[Y]").astype(int) + 1970
-    out = current_year - geburtsjahr < next(
-        iter(elterngeld_params["geschw_bonus_altersgrenzen_kinder"].keys())
+    return (
+        elterngeld_claimed
+        and arbeitsstunden_w <= elterngeld_params["max_arbeitsstunden_w"]
+        and vorjahr_einkommen_unter_bezugsgrenze
+        and kind_anspruchsberechtigt_fg
+        and monate_elterngeldbezug_unter_grenze_fg
     )
-    return out
 
 
-def elterngeld_vorschulkind(
-    geburtsjahr: int,
+def monate_elterngeldbezug_unter_grenze_fg(
+    monate_elterngeldbezug_fg: int,
+    alleinerz: bool,
+    elterngeld_anzahl_claims_fg: int,
     elterngeld_params: dict,
 ) -> bool:
-    """Check for sibling bonus on parental leave benefit.
+    """Elterngeld has been claimed for less than the maximum number of months in the
+    past.
 
     Parameters
     ----------
-    geburtsjahr
-        See basic input variable :ref:`geburtsjahr <geburtsjahr>`.
+    monate_elterngeldbezug_fg
+        See :func:`monate_elterngeldbezug_fg`.
+    alleinerz
+        See basic input variable :ref:`alleinerz <alleinerz>`.
+    elterngeld_anzahl_claims_fg
+        See :func:`elterngeld_anzahl_claims_fg`.
     elterngeld_params
         See params documentation :ref:`elterngeld_params <elterngeld_params>`.
 
     Returns
     -------
 
     """
-    current_year = elterngeld_params["datum"].astype("datetime64[Y]").astype(int) + 1970
-    out = (
-        current_year - geburtsjahr
-        < list(elterngeld_params["geschw_bonus_altersgrenzen_kinder"].keys())[1]
-    )
+    if alleinerz:
+        out = monate_elterngeldbezug_fg <= elterngeld_params["max_monate_ind"]
```

Comment:
I don't find it in the law, but the [official homepage](https://www.bmfsfj.de/bmfsfj/themen/familie/familienleistungen/elterngeld/elterngeld-73752) claims that alleinerziehend should be elterngeld_params["max_monate_paar"].